Assets/Scripts/BulletMovement.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BulletMovement : MonoBehaviour
+{
+    [SerializeField]
+    private float speed = 10;
+    [SerializeField]
+    // is the time of life until it dies
+    private float lifeSpan = 3;
+
+    void Start()
+    {   //Destroys the game object in the time of the lifeSpan
+        Destroy(gameObject, lifeSpan);
+    }
+
+    // Update is called once per frame
+    void Update()
+    {   // shoots the bullet forward
+        transform.Translate(Vector3.forward * speed * Time.deltaTime);
+
+    }
+
+    void OnTriggerEnter(Collider col)
```

Comment:
Vergeet geen access modifiers aan te geven.

In dit geval **private**

Reply:
Comment ook je functie, wat gaat de functie precies doen / waar dient de functie voor?